calculator.js

```diff
@@ -0,0 +1,82 @@
+(function() {
+    "use strict";
+
+    var result = 0;
+    var screen = new Screen(document.getElementById('screen'));
+    var buttons = document.querySelector('.allButtons');
+    var numberButtons = buttons.querySelectorAll('.number');
+
+    for (var i = 0; i <= numberButtons.length - 1; i++) {
+
+        numberButtons[i].onclick = function() {
+            var buttonDigit = parseInt(this.innerHTML);
+            screen.addDigit(buttonDigit);
+        }
+    };
+
+    var operationButtons = buttons.querySelectorAll('.operation');
+
+    for (var i = 0; i <= operationButtons.length - 1; i++) {
+        var isInitialAction = true;
+        var prevOperation;
+
+        operationButtons[i].onclick = function() {
+            var currNumber = screen.getNumber();
+
+            if (isInitialAction) {
+                result = currNumber;
+                isInitialAction = false;
+
+            } else {
+
+                switch (prevOperation) {
+
+                    case '+':
+                        result = Number(result) + Number(currNumber);
+                        break;
+
+                    case '-':
+                        result = Number(result) - Number(currNumber);
+                        break;
+
+                    case 'x':
+                        result = Number(result) * Number(currNumber);
+                        break;
+
+                    case '/':
+                        result = Number(result) / Number(currNumber);
+                        break;
+                }
+
+            }
+            screen.resetOnNextInput();
+            screen.setNumber(result);
+            prevOperation = this.innerHTML;
+        }
+    }
+
+    function Watch(fabric) {
+        this.fabric = fabric;
+    }
+    provide('which.fabric', Watch);
```

Comment:
Remove all necessary code which you put for debugging, lets return to our original calculator now when we have namespaces.